starter-code/src/components/FoodItem.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import BulmaButton from "./BulmaButton";
+
+export default class FoodItem extends Component {
+  handleDelete(e){
+    e.stopPropagation();
+    this.props.onDelete(this.props.idx);
+  }
+  render() {
```

Comment:
dentro de render... destructuring!!! `const {quantity, name, calories} = this.props`